ansible_runner/runner_config.py

```diff
@@ -393,7 +418,11 @@ def generate_ansible_command(self):
             exec_list.append(self.limit)
 
         if self.loader.isfile('env/extravars'):
-            exec_list.extend(['-e', '@{}'.format(self.loader.abspath('env/extravars'))])
+            if self.containerized:
+                extravars_path = '/runner/env/extravars'
```

Comment:
This hard-coding doesn't bring me joy 😆 

We should be able to discover the intended base. I'd like to not assume that it would *always* be `/runner` when running containerized.

Reply:
@matburt - I was looking at this yesterday. We're mapping the private data dir on the host to `/runner` in the container, which is what allows us to hardcode `/runner` as the base. That _seems_ like it should be okay, but maybe that's not actually functionally equivalent. Let me know if you would still consider `/runner` to be hard mapping given the volume mapping.

Reply:
This *is* the private data dir, though and it should be set/fetched from that:

https://github.com/ansible/ansible-runner/blob/devel/ansible_runner/runner_config.py#L88

Reply:
Actually, I retract that... because I see the attachment locality of this now. I'm still not sure how I feel about this in the general sense and it might should be based on where we know we are attaching volumes if the person is doing something *other* than what we expected them to do.